components/widgets/menu/menu_wrapper_animation.tsx

```diff
@@ -14,50 +13,27 @@ type Props = {
     show: boolean;
 }
 
-export default class MenuWrapperAnimation extends React.PureComponent<Props> {
-    private onEntering = (node: HTMLElement) => {
-        const nodeStyler = styler(node);
-        chain(
-            action(({update, complete}) => {
-                update({display: 'block'});
-                complete();
-            }),
-            tween({from: {opacity: 0}, to: {opacity: 1}, duration: ANIMATION_DURATION}),
-        ).start(nodeStyler.set);
-    }
+export default function MenuWrapperAnimation(props: Props) {
+    if (isMobile()) {
+        if (props.show) {
+            return props.children;
+        }
 
-    private onExiting = (node: HTMLElement) => {
-        const nodeStyler = styler(node);
-        chain(
-            tween({from: {opacity: 1}, to: {opacity: 0}, duration: ANIMATION_DURATION}),
-            action(({update, complete}) => {
-                update({display: 'none'});
-                complete();
-            }),
-        ).start(nodeStyler.set);
+        return null;
     }
 
-    public render() {
-        return (
-            <CSSTransition
-                in={this.props.show}
-                classNames='MenuWrapperAnimation'
-                enter={true}
-                exit={true}
-                mountOnEnter={true}
-                unmountOnExit={true}
-                {...(isMobile() ? {} : {
-                    onEntering: this.onEntering,
-                    onExiting: this.onExiting,
-                })}
-                timeout={{
-                    enter: ANIMATION_DURATION,
-                    exit: ANIMATION_DURATION,
-                }}
-            >
-                {this.props.children}
-            </CSSTransition>
-        );
-    }
+    return (
+        <CSSTransition
```

Comment:
The previous CSSTransition component wasn't actually doing anything except calling `onEntering` and `onExiting` when `props.show` changed. Now, it's doing all the work using the `MenuWrapperAnimation` CSS classes